Cargo.toml

```
@@ -31,7 +31,7 @@ az = "1.1"
 fixed = { version = "0.5.7", optional = true, default-features = false }
 float-cmp = "0.8.0"
 micromath = { version = "1.1.0", default-features = false }
-embedded-graphics-core = { path = "core", version = "^0.3.0"}
+embedded-graphics-core = { path = "core", version = "^0.3.2"}
```

Caret requirements are the default:
```suggestion
embedded-graphics-core = { path = "core", version = "0.3.2"}
```

---

Thanks, that's cleaner